packages/expo-blur/src/BlurView.tsx

```diff
@@ -63,7 +73,10 @@ class BlurView extends React.Component<BlurViewProps & BlurViewForwardedRefProp>
         <NativeBlurView
           ref={this.blurViewRef}
           tint={tint}
+          // Android uses this prop instead of the `tint`
+          tintColor={getAndroidTintColor(intensity, tint)}
```

Comment:
```suggestion
          tint={Platform.OS === "android" ? getAndroidTintColor(intensity, tint) : tint}
```
You don't need to create a separate prop, just map the tint color to a different value on Android.